### PR TITLE
GEOS-7312 - close inputstream after finishing with it

### PR DESCRIPTION
**Pull request by jonescc**
Fixes issue with RawData input streams not being closed when encoding RawData

### CODE REVIEW

File: src/extension/wps/wps-core/src/test/java/org/geoserver/wps/ppio/RawDataPPIOTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package org.geoserver.wps.ppio;
```
Comment (aaime) [inline src/extension/wps/wps-core/src/test/java/org/geoserver/wps/ppio/RawDataPPIOTest.java]: Lacks copyright header

**Comment by aaime** [comment]
Looks good, there is just a minor issue with copyright headers, then we can merge it.

**Comment by jonescc** [comment]
Copyright header added.

**Comment by aaime** [comment]
Backported to 2.8.x and 2.7.x, ticket marked as resolved

**Pull request merged**